opentsdb/opentsdb.go

```diff
@@ -252,10 +252,10 @@ func (s *Server) HandleTelnet(conn net.Conn) {
  }
 */
 type tsdbDP struct {
-	Metric    string            `json:"metric"`
-	Timestamp int64             `json:"timestamp"`
-	Value     float64           `json:"value"`
-	Tags      map[string]string `json:"tags,omitempty"`
+	Metric string            `json:"metric"`
+	Time   int64             `json:"time"`
```

Comment:
i'm not familiar with opentsdb.  did i do the right thing here or is the json going to be `"timestamp"` because it's not something we control?